ts/localtunneldebugger/v1.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Supports working with the Debug (Redirect locally) command.
+ */
+export interface LocalTunnelDebuggerV1 {
+    /**
+     * Registers a local tunnel debugger. The debugger will be invoked on user action
+     * when the user selects the Debug (Redirect locally) command in the cluster explorer,
```

Comment:
Nit: please consider renaming these `Redirect locally` in comments as well.

Reply:
Thanks for catching that.